webview/src/webview_osx.mm

```diff
@@ -0,0 +1,389 @@
+#if defined(DM_PLATFORM_OSX)
+/* NOTE:  
```

Comment:
As far as I can tell the only difference from IOS is the use of m_AllowSiteAction. The rest of the file is identical?